kernel/GBEngine/kutil.cc

```diff
@@ -9337,7 +9346,15 @@ void enterT(LObject &p, kStrategy strat, int atT)
 {
   int i;
 
-  pp_Test(p.p, currRing, p.tailRing);
+#ifdef PDEBUG
+  if (currRing->isLPring && p.shift > 0) {
+    // in this case, the order is not correct. test LM and tail separately
+    p_LmTest(p.p, currRing);
```

Comment:
Bei den Polynomen in T wird nur das Leitmonom geshiftet, deswegen ist das Leitmonom hier von geringster Ordnung.